routers/stream_router.py

```
@@ -1,11 +1,38 @@
+@router.post("/ws")
+async def receive_stream(websocket:WebSocket):
+    await websocket.accept()
+    print("EMG websocket connected")
+    
+    try:
+        while True:
+            raw_data = await websocket.receive_json()
+
+            try:
+                request = EmgWindowRequest(**raw_data)
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)